src/components/nodes-table/ColumnFilterDate.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-card-text>
+    <v-row>
+      <v-col>
+        <v-text-field
+          type="datetime-local"
+          label="From"
+          v-model="value.from"
+          clearable
+          @change="$emit('change', value)"
+        ></v-text-field>
+        <v-text-field
+          type="datetime-local"
+          label="Until"
```

Comment:
Usually in UI we use `from` and `to` (not `until`)

Reply:
Done.